src/scxml_sm_interface.cpp

```diff
@@ -46,8 +47,10 @@ static void getStateTransitionsRecursive(tinyxml2::XMLElement* state,
       throw std::runtime_error("'" + std::string(TRANSITION_ELEMENT) + "' element does not have '" +
                                std::string(EVENT_ATTRIBUTE) + "' attribute");
 
+    inherited_events.emplace(QString(event), QString(transition->Attributie(TARGET_ATTRIBUTE)));
```

Comment:
Seems like a misspelling

```suggestion
    inherited_events.emplace(QString(event), QString(transition->Attribute(TARGET_ATTRIBUTE)));
```